docs/src/index.md

```diff
@@ -1,160 +1,263 @@
-model. We first define the SIR reaction model using Catalyst
-```@example ind1
-using Catalyst
-rn = @reaction_network begin
-    α, S + I --> 2I
-    β, I --> R
+A more throughout guide for setting up Catalyst and installing Julia packages can be found [here](@ref catalyst_for_new_julia_users_packages).
+
+## [Illustrative example](@id doc_index_example)
+
+#### [Deterministic ODE simulation of Michaelis-Menten enzyme kinetics](@id doc_index_example_ode)
+Here we show a simple example where a model is created using the Catalyst DSL, and then simulated as 
+an ordinary differential equation.
+
+```@example home_simple_example
+# Fetch required packages.
+using Catalyst, OrdinaryDiffEq, Plots
+
+# Create model.
+model = @reaction_network begin
+    kB, S + E --> SE
+    kD, SE --> S + E
+    kP, SE --> P + E
 end
+
+# Create an ODE that can be simulated.
+u0 = [:S => 50.0, :E => 10.0, :SE => 0.0, :P => 0.0]
+tspan = (0., 200.)
+ps = (:kB => 0.01, :kD => 0.1, :kP => 0.1)
+ode = ODEProblem(model, u0, tspan, ps)
+
+# Simulate ODE and plot results.
+sol = solve(ode)
+plot(sol; lw = 5)
 ```
-Assuming [Graphviz](https://graphviz.org/) and is installed and *command line
-accessible*, the network can be visualized using the [`Graph`](@ref) command
-```julia
-Graph(rn)
-```
-which in Jupyter notebooks will give the figure
 
-![SIR Network Graph](assets/SIR_rn.svg)
+#### [Stochastic jump simulations](@id doc_index_example_jump)
+The same model can be used as input to other types of simulations. E.g. here we instead perform a 
+jump simulation
+```@example home_simple_example
+# Create and simulate a jump process (here using Gillespie's direct algorithm).
+# Note that integer (not decimal) initial conditions are used.
+using JumpProcesses
+u0_integers = [:S => 50, :E => 10, :SE => 0, :P => 0]
+dprob = DiscreteProblem(model, u0_integers, tspan, ps)
+jprob = JumpProblem(model, dprob, Direct())
+jump_sol = solve(jprob, SSAStepper())
+jump_sol = solve(jprob, SSAStepper(); seed = 1234) # hide
+plot(jump_sol; lw = 2)
+```
 
-To generate and solve a mass action ODE version of the model we use
-```@example ind1
-using OrdinaryDiffEq
-p     = [:α => .1/1000, :β => .01]
-tspan = (0.0,250.0)
-u0    = [:S => 999.0, :I => 1.0, :R => 0.0]
-op    = ODEProblem(rn, u0, tspan, p)
-sol   = solve(op, Tsit5())       # use Tsit5 ODE solver
+## [Elaborate example](@id doc_index_elaborate_example)
+In the above example, we used basic Catalyst-based workflows to simulate a simple model. Here we 
+instead show how various Catalyst features can compose to create a much more advanced model. Our 
+model describes how the volume of a cell ($V$) is affected by a growth factor ($G$). The growth 
+factor only promotes growth while in its phosphorylated form ($Gᴾ$). The phosphorylation of $G$ 
+($G \to Gᴾ$) is promoted by sunlight (modeled as the cyclic sinusoid $kₐ*(sin(t)+1)$), which
+phosphorylates the growth factor (producing $Gᴾ$). When the cell reaches a critical volume ($Vₘ$)
```

Comment:
Maybe just use `Gp` or a subscript of `p` instead of Gᴾ? The latter looks like an exponent to me which might be a little confusing?